website/docs/r/vpc_endpoint_service.html.markdown

```diff
@@ -54,15 +54,15 @@ The following arguments are supported:
 
 In addition to all arguments above, the following attributes are exported:
 
-* `id` - The ID of the VPC endpoint service.
-* `availability_zones` - The Availability Zones in which the service is available.
-* `arn` - The Amazon Resource Name (ARN) of the VPC endpoint service.
-* `base_endpoint_dns_names` - The DNS names for the service.
-* `manages_vpc_endpoints` - Whether or not the service manages its VPC endpoints - `true` or `false`.
-* `private_dns_name` - The private DNS name for the service.
-* `service_name` - The service name.
-* `service_type` - The service type, `Gateway` or `Interface`.
-* `state` - The state of the VPC endpoint service.
+* `id` - string - The ID of the VPC endpoint service.
+* `availability_zones` - set(string) - The Availability Zones in which the service is available.
+* `arn` - string - The Amazon Resource Name (ARN) of the VPC endpoint service.
+* `base_endpoint_dns_names` - set(string) - The DNS names for the service.
+* `manages_vpc_endpoints` - bool - Whether or not the service manages its VPC endpoints - `true` or `false`.
+* `private_dns_name` - string - The private DNS name for the service.
+* `service_name` - string - The service name.
+* `service_type` - string - The service type, `Gateway` or `Interface`.
+* `state` - string - The state of the VPC endpoint service.
```

Comment:
Hi @daisakuhazui! Until we have a decision on #15365, lets keep with the standard that is most consistent across the provider and just mention Set in the description.

```suggestion
* `id` - The ID of the VPC endpoint service.
* `availability_zones` - A set of Availability Zones in which the service is available.
* `arn` - The Amazon Resource Name (ARN) of the VPC endpoint service.
* `base_endpoint_dns_names` - A set of DNS names for the service.
* `manages_vpc_endpoints` -  Whether or not the service manages its VPC endpoints - `true` or `false`.
* `private_dns_name` - The private DNS name for the service.
* `service_name` - The service name.
* `service_type` - The service type, `Gateway` or `Interface`.
* `state` - The state of the VPC endpoint service.
```